kernel/rtlil.h

```diff
@@ -760,6 +761,8 @@ struct RTLIL::SigBit
 	SigBit(const RTLIL::SigBit &sigbit) = default;
 	RTLIL::SigBit &operator =(const RTLIL::SigBit &other) = default;
 
+	inline int is_wire() const { return wire != NULL; }
```

Comment:
Arguably these should return `bool` rather than `int`. This may also give you an excuse to update the branch, which will hopefully not encounter the same failure when the tests are re-run.